src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -318,10 +316,10 @@
   []
```

Comment:
Also, none of these `query-rec`'s need to be functions now that we're always using postgresql.